app/src/androidTest/java/org/simple/clinic/reports/ReportsSyncAndroidTest.kt

```diff
@@ -0,0 +1,65 @@
+package org.simple.clinic.reports
+
+import androidx.test.runner.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.simple.clinic.AuthenticationRule
+import org.simple.clinic.TestClinicApp
+import org.simple.clinic.util.Just
+import org.simple.clinic.util.None
+import org.simple.clinic.util.RxErrorsRule
+import java.io.File
+import javax.inject.Inject
+
+@RunWith(AndroidJUnit4::class)
+class ReportsSyncAndroidTest {
+
+  @Inject
+  lateinit var reportsRepository: ReportsRepository
+
+  @Inject
+  lateinit var reportsSync: ReportsSync
+
+  private val authenticationRule = AuthenticationRule()
+
+  private val rxErrorsRule = RxErrorsRule()
+
+  @get:Rule
+  val ruleChain = RuleChain
+      .outerRule(authenticationRule)
+      .around(rxErrorsRule)!!
```

Comment:
Can we instantiate both rules here instead of keeping them as class fields? I realize we've been doing this in other places and was copy pasted here.

Reply:
Done.